parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoSchemaConverter.java

```diff
@@ -98,17 +117,18 @@ public MessageType convert(Class<? extends Message> protobufClass) {
                                                                                                    OriginalType originalType,
                                                                                                    final GroupBuilder<T> builder) {
     return builder
-        .group(Type.Repetition.REQUIRED).as(OriginalType.LIST)
+        .group(Type.Repetition.OPTIONAL).as(OriginalType.LIST)
```

Comment:
@BenoitHanotte, we had a discussion a while back around the LIST wrapper being REQUIRED:
https://github.com/apache/parquet-mr/pull/411#issuecomment-301194565

Reply:
I strongly believe the wrappers should be optional for the following reasons:
1. it is consistent with how Maps are handled
2. it is consistent with the fact that parquet-protobuf does not write fields if they have the default values (protobuf considers them cleared). this is currently the case for primitives and complex messages, I would like collections to behave the same way.
3. every field is optional with protobuf 3
I haven't seen any arguments for making the wrapper required in the first place, was there a specific reason I am not aware of?

Reply:
Please note than in my comment above I say "*If* you want to have systematically a 3 level ..."
I think in protobuf you don't want to have it systematically because the parquet schema maps to the proto schema.